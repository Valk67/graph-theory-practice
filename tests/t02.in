9 11
0 1 0
0 4 0
1 2 0
2 4 0
4 5 0
4 7 0
5 3 0
3 0 0
7 6 0
3 6 0
7 8 0
