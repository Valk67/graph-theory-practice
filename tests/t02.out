Testing BFS on node 0.
0 1 4 2 5 7 3 6 8 
Testing print path on node 3.
0 4 5 3 
Testing print levels on node 0.
0 
1 4 
2 5 7 
3 6 8 
Testing DFS.
0 1 2 4 5 3 6 7 8 
Testing to check if a graph has a cycle.
Graph has a cycle.
